pkg/providers/instancetype/suite_test.go

```diff
@@ -177,10 +177,12 @@ var _ = Describe("Instance Types", func() {
 			"beta.kubernetes.io/os":         "linux",
 			v1.LabelInstanceType:            "g4dn.8xlarge",
 			"topology.ebs.csi.aws.com/zone": "test-zone-1a",
+			v1.LabelWindowsBuild:            "some-build",
 		}
 
+		expected := v1alpha5.WellKnownLabels
```

Comment:
nit: any reason to re-label this? Why not just leave this as-is?

Reply:
You are right.
I should revert to its original version and only add windows build in the above code.